### ISSUE TITLE
supports_masking is missing from keras.layers.Layer class

### ISSUE THREAD

**Issue by lostmsu** [losttech/Gradient#4]
Subj.

@AndreLuettig

**Comment by lostmsu** [comment]
Fixed in Preview 5.1
